R/pkg/R/mllib_tree.R

```diff
@@ -374,6 +374,10 @@ setMethod("write.ml", signature(object = "GBTClassificationModel", path = "chara
 #'                     nodes. If TRUE, the algorithm will cache node IDs for each instance. Caching
 #'                     can speed up training of deeper trees. Users can set how often should the
 #'                     cache be checkpointed or disable it by setting checkpointInterval.
+#' @param handleInvalid How to handle invalid data (unseen labels or NULL values) in classification model.
+#'        Supported options: "skip" (filter out rows with invalid data),
+#'                           "error" (throw an error), "keep" (put invalid data in a special additional
```

Comment:
is "error" the default behavior? since we are doing `handleInvalid = c("error", "skip", "keep")` could you add text to say it's the default. also consider sorting the options - default first "error", then "keep", "skip" if we are alphabetical?